### PR TITLE
Improved candidate experence

### PR DESCRIPTION
**Pull request by Eunoia**
* Removed gems that could cause issues when installing rails
* Edited for tone and grammar.
* Included database to make setup easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,50 +48,48 @@ You can find the stylesheet file in app/assets/stylesheets/home.css
 
 3. Run "bundle install" to install Rails and all it's dependencies.
 
-4. Set up the database with "rake db:create" and then "rake db:migrate" and then "rake db:seed"
-
 5. Start your rails app with "rails s"
 
-6. Navigate to http://localhost:3000 and view the Nile application in all it's glory. You can also test to make sure the backend is working by navigating to /by_title, /by_author, /by_subject to make sure data is being returned.
+6. Navigate to <http://localhost:3000> and view the Nile application in all it's glory. You can also test to make sure the backend is working by navigating to <http://localhost:300/by_title>, <http://localhost:300/by_author>, <http://localhost:300/by_subject> to see the data being retuned.
```
Comment (MikeAbner) [inline README.md]: these need to be 3000 not 300

File: README.md [hunk 2]
```diff
@@ -46,50 +48,48 @@ You can find the stylesheet file in app/assets/stylesheets/home.css
...
+3. The search box does not work! It's supposed to execute a search when the user enters the correct search request and presses enter.  We don't want any buttons on this page, so make sure the search box can respond to the enter key being pressed. When that happens you should call the appropriate endpoint and display the results of the search.
 
-4. After every search is complete, you should add what was searched to the search results sections. Each entry in that section should also list the number of search results returned for that search. You should allow the user to click on that item to re-execute the search.
+4. After every search is complete, you should add what was searched to the search results sections. Each entry in that section should also list the number of search results returned for that search. You should allow the user to click on that item to re-execute the search. You don't need to persist the history.
```
Comment (MikeAbner) [inline README.md]: You don't need to persist the history to the backend.

File: README.md [hunk 3]
```diff
@@ -46,50 +48,48 @@ You can find the stylesheet file in app/assets/stylesheets/home.css
...
 
 2. Does the Nile application work the way it should based on the Requirements section?
 
 3. Is the code well organized and does it follow traditional javascript coding conventions?
 
-4. Bells and Whistles. If there is time, what were you able to add. For example, the fonts in the mock are located here: https://fonts.google.com/specimen/Noto+Sans and http://fontawesome.io/assets/font-awesome-4.7.0.zip.
+4. Bells and Whistles. If there is time, how can you improve on the bookstore? For example, the fonts in the mock are located here: https://fonts.google.com/specimen/Noto+Sans and http://fontawesome.io/assets/font-awesome-4.7.0.zip. 
 
-5. Your summary statement. Do your choices make sense? Do you explain them well? Are we left with a lot of open questions?
+5. Your summary statement. Are your choices sensible, and well explained?
```
Comment (MikeAbner) [inline README.md:89]: remove the comma

File: app/models/search.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 class Search
 
   def execute type, term
-    puts "TYPE: '#{type}'"
+    puts "TYPE: '#{type}', term: #{term}"
```
Comment (MikeAbner) [inline app/models/search.rb:4]: I think we should remove this...the app should be returning a good json error message with this data in it if it doesn't match one of the types.

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -18,6 +19,5 @@
       </footer>
     </div>
   </div>
-  <%= javascript_include_tag "//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js" %>
```
Comment (MikeAbner) [inline app/views/layouts/application.html.erb:21]: This is the correct place to put this code...

**Review by MikeAbner — CHANGES_REQUESTED** [review]
Couple of things. But thanks for removing all that crap. He should never have to look outside the app/assets or app/views directory, so would never have seen this stuff, but it'll make it more clear if he's poking around.

File: app/models/search.rb (hunk 4 above)
Comment (Eunoia) [reply app/models/search.rb:4]: It should, but just in case.

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (Eunoia) [reply app/views/layouts/application.html.erb:21]: The application.js loads before this line, resulting in an error that requires playing in application.html.erb

File: README.md (hunk 3 above)
Comment (Eunoia) [reply README.md:89]: "well explained" is an appositive for choices. Commas go before an appositive. https://www.scribendi.com/advice/what_are_appositives.en.html

File: README.md (hunk 2 above)
Comment (Eunoia) [reply README.md]: The history doesn't need to be persisted between page reloads.

File: README.md (hunk 1 above)
Comment (Eunoia) [reply README.md]: Good point.

**Pull request merged**